local DialogueData = {
	["Intro"] = {
		"You find yourself in a an unfamiliar Stone Fortress..",
		"Only the fire that you have lit is the clue that this is the right path towards freedom..",
		"The fire is the only thing that keeps you moving forward.."
	},

	["ArcaneKnowledge"] = {
		"You stumble upon a ritual which took place near the prisoner torture chamber..",
		"You quickly flip through the pages and learn what was your purpose here as a prisoner",
		"You were supposed to serve him, The Great Tyrant..",
		"You set out on a mission to find the one responsible for all of this.."
	},

	-- Zde si můžeš přidat další: ["ID"] = {"Text 1", "Text 2"},
}

return DialogueData
