local DialogueData = {
	["Intro"] = {
		"You wake up inside your rotting cell but something seems off..",
		"You quickly notice that the guards and all the prisoners were massacred by something unknown..",
		"All thats left for you is to escape this damned Dungeon."
	},

	["ArcaneKnowledge"] = {
		"You stumble upon a ritual which took place near the prisoner torture chamber..",
		"You quickly flip through the pages and learn what was your purpose here as a prisoner",
		"You were supposed to serve him, The Great Tyrant..",
		"You set out on a mission to find the one responsible for all of this.."
	},

	-- Zde si můžeš přidat další: ["ID"] = {"Text 1", "Text 2"},
}

return DialogueData
